operations/availabilityblocks.md

```diff
@@ -66,220 +68,188 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
-| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). If not specified, the operation returns data for all enterprises within scope of the Access Token. |
-| `AvailabilityBlockIds` | string | optional, max 1000 items | Unique identifiers of the requested [Availability blocks](#availability-block). |
-| `ServiceIds` | string | optional, max 1000 items | Unique identifiers of the [Services](services.md#service) to which [Availability blocks](#availability-block) are assigned. |
-| `CreatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Availability blocks](#availability-block) were created. |
-| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Availability blocks](#availability-block) were updated. |
-| `CollidingUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Availability blocks](#availability-block) are active. |
-| `ReleasedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Availability blocks](#availability-block) are released. |
-| `States` | array of string [Availability block state](#availability-block-state) | optional | States the availability blocks should be in. |
-| `ExternalIdentifiers` | string | optional, max 1000 items | Identifiers of [Availability block](#availability-block)s from external systems. |
-| `ActivityStates` | array of string [Activity state](_objects.md#activity-state) | optional | Whether to return only active, only deleted or both records. |
-| `Extent` | [Availability block extent](#availability-block-extent) | required | Extent of data to be returned, e.g. it is possible to specify that related service orders (for example reservations) are returned. |
-
-#### Availability block state
-
-* `Confirmed` - The block deducts availability and can have reservations assigned.
-* `Optional` - The block deducts availability and cannot have reservations assigned.
-* `Inquired` - The block does not deduct availability and cannot have reservations assigned \(waitlist\).
-* `Released` - The block deducts availability, but only for reservations assigned to the block before release. Remaining availability is returned back to general availability \(waitlist\).
-* `Canceled` - The block does not deduct availability and cannot have reservations assigned \(waitlist\). 
+| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the Enterprises. If not specified, the operation returns data for all enterprises within scope of the Access Token. |
+| `Extent` | [Availability block extent](availabilityblocks.md#availability-block-extent) | required | Extent of data to be returned, e.g. it is possible to specify that related service orders (for example reservations) are returned. |
```

Comment:
I believe this one is not required and has default when it's not gonna return anything 😅 Verify it but I would make it reflect actual reality so to avoid confusion.

Reply:
Agreed, all extents are optional by default.